src/XProfile/VirtoCommerce.XProfile/Schemas/ContactType.cs

```diff
@@ -19,7 +20,11 @@ public class ContactType : ExtendableGraphType<ContactAggregate>
     {
         private readonly IOrganizationAggregateRepository _organizationAggregateRepository;
 
-        public ContactType(IOrganizationAggregateRepository organizationAggregateRepository, IDynamicPropertyResolverService dynamicPropertyResolverService)
+        public ContactType(
+            IOrganizationAggregateRepository organizationAggregateRepository,
+            IDynamicPropertyResolverService dynamicPropertyResolverService,
+            IMediator mediator,

```

Comment:
@Vectorfield4 need to add info about this change to the release note.

Reply:
@Vectorfield4 you didn't :)